docs/source/index.rst

```diff
@@ -1,65 +1,139 @@
-=====
-Conda
-=====
-
-.. figure::  /img/conda_logo.svg
-   :align: center
-   :width: 50%
-
-   ..
-
-|
-
-:emphasis:`Package, dependency and environment management for any
-language---Python, R, Ruby, Lua, Scala, Java, JavaScript, C/ C++,
-FORTRAN`
-
-Conda is an open-source package management system and environment
-management system that runs on Windows, macOS, and Linux. Conda
-quickly installs, runs, and updates packages and their dependencies.
-Conda easily creates, saves, loads, and switches between environments
-on your local computer. It was created for Python programs but it
-can package and distribute software for any language.
-
-Conda as a package manager helps you find and install packages.
-If you need a package that requires a different version of
-Python, you do not need to switch to a different environment
-manager because conda is also an environment manager. With just
-a few commands, you can set up a totally separate environment to
-run that different version of Python, while continuing to run
-your usual version of Python in your normal environment.
-
-In its default configuration, conda can install and manage the
-over 7,500 packages at repo.anaconda.com that are built, reviewed,
-and maintained by Anaconda\ |reg|.
-
-Conda can be combined with continuous integration systems, such
-as GitHub Actions, to provide frequent, automated testing
-of your code.
-
-The conda package and environment manager is included in all versions of
-:ref:`Anaconda <anaconda-glossary>`\ |reg|,
-:ref:`Miniconda <miniconda-glossary>`, and
-`Anaconda Repository <https://docs.continuum.io/anaconda-repository/>`_.
-Conda is also included in `Anaconda Enterprise
-<https://www.anaconda.com/enterprise/>`_, which provides on-site enterprise
-package and environment management for Python, R, Node.js, Java, and other
-application stacks. Conda is also available on
-`conda-forge <https://anaconda.org/conda-forge/conda>`_, a community channel.
-You may also get conda on `PyPI <https://pypi.org/>`_, but
-that approach may not be as up to date.
+===================
+Conda Documentation
+===================
+
+Welcome to conda's documentation! Conda provides package, dependency and environment
+management for any language. Here, you will find everything you need to get started
+using conda in your own projects.
+
+Install  :octicon:`download;1em;sd-text-primary`
+................................................
+
+We recommend the following methods to install conda:
```

Comment:
Let's also including instructions for:
- Homebrew:
    ```
    brew install miniconda
    ```
- Chocolatey
    ```
    choco install miniconda3
    ```

Is there a way to make these tabs more similar? I think MinIO's download page is a great example of what I hope this can look like

Reply:
@kenodegard,

That's a great example! I love those icons 😍. I'll see what I can come up with.

Also, yes on adding brew and chocolatey. That makes sense.

Reply:
@kenodegard,

I added the two new tabs you requested plus some icons for each.

I am wondering what you mean by "more similar". Care to elaborate? 🤔

Reply:
@kenodegard,

I think I have them looking more similar now. Please take a look when you get a chance.

Reply:
yeah i wasn't a fan of how much the text and look of each tab changed as you clicked through them, yes the current version looks much better

can we add a divider between the two macOS sections:
<img width="747" alt="Screenshot 2023-11-14 at 9 14 56 AM" src="https://github.com/conda/conda/assets/4546435/4042b7bc-5584-42a7-bb3c-236cb52e45b3">

also for the icons, since neither homebrew nor chocolatey have proper icons maybe just use the icons for the platforms they are for, so macOS and Linux for homebrew and windows for chocolatey?

<img width="659" alt="Screenshot 2023-11-14 at 9 24 35 AM" src="https://github.com/conda/conda/assets/4546435/7f84f821-2a89-4315-8fb5-58476a055ca2">

Reply:
Alternatively, what are your thoughts on combining the bold text with the download:
<img width="231" alt="Screenshot 2023-11-14 at 11 00 48 AM" src="https://github.com/conda/conda/assets/4546435/68ae1bfc-70d9-47db-a54c-9d587c3093b4">
<img width="650" alt="Screenshot 2023-11-14 at 11 00 51 AM" src="https://github.com/conda/conda/assets/4546435/5a627c27-ebec-411e-a081-d171b7e02c4d">

Reply:
I like putting the name of the install on the button itself. Less text is better 😄 .

Reply:
Have you double-checked with Marco if we can link to Miniconda via Chocolatey and Homebrew here? Both are _not_ maintained by Anaconda, despite Miniconda being an Anaconda product.